src/Console/Command/PurgeCommand.php

```diff
@@ -39,8 +39,9 @@ protected function configure()
 newest json file in the include directory of the given output-dir.
 
 In your satis.json (or other name you give), you must define
-"archive" argument.
+"archive" argument. You also need to define "homepage"* if you don't use archive "prefix-url".
```

Comment:
Is the extra leading `*` after `"homepage"` intentional ? :thinking:

Reply:
yes, it points to 2 lines below: ´* "homepage" may be overloaded using SATIS_HOMEPAGE environment variable.´

could probably be rewritten to `You also need to define "homepage" argument or "SATIS_HOMEPAGE" environment variable, if you don't use archive "prefix-url".`

and then drop the line below.

Reply:
Sounds clearer indeed :wink: :rocket: